src/volumeindicatoradapter/DefaultVolumeIndicatorAdapter.ts

```diff
@@ -85,6 +101,35 @@ export default class DefaultVolumeIndicatorAdapter implements VolumeIndicatorAda
         }
       }
     }
+    if (this.sessionReconnected) {
+      this.cleanUpState(info);
+      this.sessionReconnected = false;
+    }
+  }
+
+  private cleanUpState(info: SdkAudioStreamIdInfoFrame): void {
+    const localAttendeeIds = Object.values(this.streamIdToAttendeeId);
+    const remoteAttendeeIds = info.streams.map(stream => stream.attendeeId);
+    const deletedAttendeeIds = localAttendeeIds.filter(id => {
+      return !remoteAttendeeIds.includes(id);
+    });
+
+    for (const [index, deletedAttendeeId] of deletedAttendeeIds.entries()) {
+      const streamId = this.attendeeIdToStreamId[deletedAttendeeId];
+      const externalUserId = this.streamIdToExternalUserId[streamId];
+      delete this.streamIdToAttendeeId[streamId];
+      delete this.streamIdToExternalUserId[streamId];
+      delete this.warnedAboutMissingStreamIdMapping[streamId];
+      delete this.attendeeIdToStreamId[deletedAttendeeId];
+
+      this.realtimeController.realtimeSetAttendeeIdPresence(
+        deletedAttendeeId,
+        false,
+        externalUserId,
+        false,
```

Comment:
Technically, we don't know whether they dropped or not. It might be worth just adding a comment to that effect that given the information in protocol, dropped/left is unknowable.

Reply:
Added comments here.